tasks/st0236_set_cache_paths_to_default.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
 # shellcheck disable=SC2230
+# DEPRECATION:
+# This script is now Deprecated and will be removed in a further update
 declare PT__installdir
 source "$PT__installdir/bash_task_helper/files/task_helper.sh"
 
 if [ -n  "$(facter -p pe_build)" ]
 then
-	task-suceed "success - Not an agent node"
+  echo "This Script is now Deprecated and has been replaced by the task-output function, which provides the same functionality"
+  echo "This task will be removed in a future release. Please see this module's README for more information"
```

Comment:
I think we should follow the approach used in the other deprecation we did, which is to include the message in the output of the task instead of just printing it.  The [task-output](https://github.com/puppetlabs/puppetlabs-bash_task_helper/#task-output) function of the helper should make this easy, and if my reading of it is correct we can just add something after the `source` on line 6 like:

```
task-output "deprecation" "This task is deprecated and will be removed in a future release. Please see this module's README for more information"
```

The message should be included whenever the task exits according to the docs.

Reply:
Alright, perhaps that's the best method from now on in keeping the repo clean.